### PR TITLE
Update the OBS project name from "OpenHPC" to "OpenHPC3"

### PR DESCRIPTION
**Pull request by martin-g**
Drop using the legacy crypto policies.

### CODE REVIEW

File: misc/obs/obs_config.py [hunk 1]
```diff
@@ -79,7 +79,7 @@ def __init__(self, version):
         logging.info("--> Micro release   = %s" % self.microVer)
 
         if self.microVer == '0':
-            self.obsProject = "OpenHPC:" + self.branchVer + ":Factory"
+            self.obsProject = "OpenHPC3:" + self.branchVer + ":Factory"
```
Comment (adrianreber) [inline misc/obs/obs_config.py]: Do not change this file, please.

Comment (martin-g) [reply misc/obs/obs_config.py]: Reverted!

**Comment by adrianreber** [comment]
Errors are expected currently because the repository moved.

**Comment by adrianreber** [comment]
The logs show that the packages signed with the new key are installed without dnf/rpm complaining. Looks good. Thanks.

**Pull request merged**